### DISCUSSION TITLE
Fixing false positives (UI indicates results found when there are none)

### DISCUSSION THREAD

**Discussion by ianray42** [Purfview/IMDb-Scout-Mod#134] [General]
First and foremost -- love the app, use it a TON in filling requests.

Some of the things I search for are pretty obscure, so I am used to getting 0 results, even with my fairly large net of selected filters.  Sometimes, I get that coveted green box and get excited, only to find out it was simply a parsing error.

~ SO ~

I started to investigate some of the errors and believe I have fixed them, so I thought it would be good to start a thread where we share the small userfixes we've discovered, potentially to be included in future releases while saving the glorious devs some time. :)

I\'ll start:
------------------------------

- **Knaben Database** (as of v17.6 of IMDb_Scout_Mod.user.js)
  - https://github.com/Purfview/IMDb-Scout-Mod/blob/762b59f88efb300137bd7dee5548e39e60637df4/IMDb_Scout_Mod.user.js#L1458
  - https://github.com/Purfview/IMDb-Scout-Mod/blob/762b59f88efb300137bd7dee5548e39e60637df4/IMDb_Scout_Mod.user.js#L1462
  - `'matchRegex': />Total hits : 0</},` --> `'matchRegex': />Total hits: 0</},` (removal of space before colon)

**Comment by Purfview** [comment]
Sites constantly change their code, so, various discrepancies arise in time.

About saving time: just do PR [Pull request] with fixes per site.

**Comment by Purfview** [comment]
To do PR is simple: Press on the script > Press Edit. 
That will create a fork for you automatically, and after your edits GitHub should ask if you want to do PR, then it should be obvious what to press next.

**Reply by ianray42** [reply]
Okay, no worries -- I will just do that.
